wordpress-mobile/WordPress-Android
# Issue/remove deprecated gif picker

This PR removes deprecated `GifPickerActivity` and all the related classes.

We were using a `KEY_SAVED_MEDIA_MODEL_LOCAL_IDS` constant from `GifPickerActivity`, which I extracted into `MediaPickerConstants`.

To test:
- `GifPickerActivity` was not in use, so there is not much to test.
- Make sure the app builds and installs.


PR submission checklist:
